File: DUMPFutsalTournament/Data/Entities/MatchEvent.cs
using DUMPFutsalTournament.Data.Enums;

namespace DUMPFutsalTournament.Data.Entities
{
    public class MatchEvent
    {
        public int MatchEventId { get; set; }
        public int EventMinute { get; set; }
        public int MatchId { get; set; }
        public Match Match { get; set; }
        public int? PlayerId { get; set; }
        public Player Player { get; set; }
        public MatchEventType EventType { get; set; }
        public bool IsForHomeTeam { get; set; }
    }
}

File: DUMPFutsalTournament/Web/src/app/infrastructure/classes/matchevent.ts
import { Match } from './match';
import { Player } from './player';
import { MatchEventType } from '../enums/matcheventtype';

export class MatchEvent {
	constructor(match: Match, player: Player, eventType: MatchEventType, isForHomeTeam: boolean, eventMinute: number) {
		this.match = match;
		this.player = player;
		this.eventType = eventType;
		this.isForHomeTeam = isForHomeTeam;
		this.eventMinute = eventMinute;
	}

	matchEventId: number;
	eventMinute: number;
	match: Match;
	player: Player;
	eventType: MatchEventType;
	isForHomeTeam: boolean;
}